packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -421,7 +419,7 @@ export class DetailsList extends React.Component<IDetailsListProps, IDetailsList
       if (this.refs.selectionZone) {
         this.refs.selectionZone.ignoreNextFocus();
       }
-      row.focus();
+      this._async.setTimeout(() => row.focus(), 0);
```

Comment:
@jijizz I'm a little afraid of this change. Sometimes delaying focus can cause unexpected consequences.

Did it improve perf in your measurements?